lib/Db/ItemMapperV2.php

```diff
@@ -190,7 +191,16 @@ public function deleteOverThreshold(int $threshold, bool $removeUnread = false):
         $deleteQb->delete($this->tableName)
                  ->where('id IN (?)');
 
-        return $this->db->executeUpdate($deleteQb->getSQL(), [$total_items], [IQueryBuilder::PARAM_INT_ARRAY]);
+        $affected_rows = 0;
+        // split $total_items into multiple chunks because PDO can't handle more than 65535 parameters
+        foreach (array_chunk($total_items, 65000) as $items_chunk) {
```

Comment:
Is there a constant for this limit?

Reply:
I don't know for sure. If there is one, I have not found it.

Reply:
Can you add it as a constant to the NewsMapper class then? Makes it easier to use in different places and allows for a nicer comment.